eng/Build.props

```diff
@@ -5,5 +5,7 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectToBuild Include="$(RepoRoot)Build.proj" />
+    <!-- dotnet/runtime controls which projects run in which passes through our DefaultSubsets settings in eng/Subsets.props -->
+    <ProjectToBuild Include="$(RepoRoot)Build.proj" BuildPass="2" />
```

Comment:
Doesn't the above unconditional ProjectToBuild item need to be conditioned on when BuildPass is not 2? Also you might want to follow the syntax that we use in aspnetcore:

https://github.com/dotnet/aspnetcore/blob/9f3e60c3cfc720f662c87e8430284c05e2e084cb/eng/Build.props#L73-L80

Reply:
We have filtering in Arcade that automatically filters ProjectToBuild items down by BuildPass. That's why we need this change at all. Without this change, Build.proj is filtered out in BuildPass2.

Reply:
Right but from what I see, this only applies to VMR inner builds: https://github.com/dotnet/arcade/blob/fbc07804abc2b4bf553057479258b8ae65df037f/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L82-L86

Wouldn't Build.proj get invoked twice on non VMR inner builds? IIRC that was the reason why we also put conditions on the items.

Reply:
Good catch. I'll add a condition on this one.